src/isaac_ros/isaac_ros_launch/package.xml

```diff
@@ -18,7 +18,7 @@
   <exec_depend>nvblox_rviz_plugin</exec_depend>
   <exec_depend>nvblox_examples_bringup</exec_depend>
   <exec_depend>nvblox_nav2</exec_depend>
-  <exec_depend>combine_esdf</exec_depend>
+  <!-- <exec_depend>combine_esdf</exec_depend> -->
```

Comment:
Can we just delete this

Reply:
it could serve as a reminder for when we add combine_esdf back later, up to @BergerKing333 though, doesn't really matter either way